File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.java
import java.util.Arrays;
class Solution {
    public String solution(String my_string) {
        char [] ch = new char[my_string.length()];
        for(int i = 0; i < my_string.length(); i++){
            if(my_string.charAt(i) < 91){
                ch[i] = (char)(my_string.charAt(i)+32);
            }
            else{
                ch[i] = my_string.charAt(i);
            }
        }
        Arrays.sort(ch);
        String answer = new String(ch);
        return answer;
    }
}

File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.cs
using System;

class Solution {
    public String solution(String my_string) {
        char [] ch = new char[my_string.Length];
        for(int i = 0; i < my_string.Length; i++){
            if(my_string[i] < 91){
                ch[i] = (char)(my_string[i]+32);
            }
            else{
                ch[i] = my_string[i];
            }
        }
        Array.Sort(ch);
        String answer = new String(ch);
        return answer;
    }
}